.github/workflows/nix.yaml

```diff
@@ -118,49 +118,41 @@ jobs:
             nixpkgs#skopeo \
             -c .github/workflows/nix.nu '${{ inputs.config }}'
       - name: Upload Tarball to Github
-        if: steps.cfg.outputs.upload-to-github == 'true'
+        if: >-
+          runner.os == 'Linux' &&
+          steps.cfg.outputs.upload-to-github == 'true'
         uses: actions/upload-artifact@v4
         with:
           name: tarball
           path: "./packages/tarball/*.tar.gz"
           if-no-files-found: ignore
       - name: Upload Debian Package to Github
-        if: steps.cfg.outputs.upload-to-github == 'true'
+        if: >-
+          runner.os == 'Linux' &&
+          steps.cfg.outputs.upload-to-github == 'true'
         uses: actions/upload-artifact@v4
         with:
           name: debian
           path: "./packages/debian/*.deb"
           if-no-files-found: ignore
```

Comment:
Shouldn't we error here now if now files are found, since we rely on the presence of `./packages/debian/*.deb` later on in the install step anyways?

Reply:
Valid point.